package.json

```diff
@@ -39,51 +39,65 @@
       "command": "babel src -d lib --watch"
     }
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "prettier --config .prettierrc --write",
+      "eslint -c .eslintrc.json --ext .jsx,.js --fix",
+      "git add"
+    ]
+  },
   "devDependencies": {
-    "@deck.gl/core": "^7.1.10",
-    "@deck.gl/mapbox": "^7.1.10",
+    "@deck.gl/core": "7.1.10",
```

Comment:
Any reason to fix the versions here and not in other cases when we have a yarn.lock?

Reply:
cause @deck.gl is not following semver